functions.opengraph.php

```diff
@@ -18,6 +18,8 @@
  * Outputs Open Graph tags generated by Jetpack.
  */
 function jetpack_og_tags() {
+	global $post;
+	$data = $post; // so that we don't accidentally explode the global.
```

Comment:
This seemed like a strange comment, so I decided to look into what might be "exploding".

I didn't see anything that would obviously "explode". I tracked the comment down to 34cf22c2827d0a8fb01c7ed1201763b603c0f373, where I think the "exploding" was probably more due to the use of `setup_postdata()` than to the non-use of a copy.

And also, now that I think about it, the making a copy here does almost nothing anyway. There's no clone happening here so `$data->foo` is going to access the same WP_Post object; the only thing this might prevent is if some code below wants to reassign `$data` itself. Or if some function called below reassigns `$post` as a side effect, but that concern wouldn't fit with "so that we don't accidentally explode the global" comment.

Reply:
I was curious as well; this thread provides more information about why this was introduced:
https://wordpress.org/support/topic/jetpack-processes-shortcodes-twice/page/2/#post-3362426